Examples/Examples/ContentView.swift

```diff
@@ -47,6 +48,24 @@ struct ContentView: View {
                         Label("Load HTML String", systemImage: "doc")
                             .labelStyle(.iconOnly)
                     }
+
+                    Button {
+                        Task {
+                            viewState.pdf = try! await proxy.contentReader.pdf()
+                        }
+                    } label: {
+                        Label("Take PDF", systemImage: "printer")
+                            .labelStyle(.iconOnly)
+                    }
+                    .accessibilityIdentifier("take_pdf_button")
+
+                    if let data = viewState.pdf,
+                       let pdf = PDFDocument(data: data) {
+                        ShareLink(item: pdf,
+                                  preview: SharePreview("PDF Document",
+                                                        image: pdf.thumbnail ?? Image(systemName: "doc.text")))
+                        .labelStyle(.iconOnly)
```

Comment:
Due to the default indentation rules, which make chained methods appear disconnected, I prefer the following indentation style.

```suggestion
                        ShareLink(
                            item: pdf,
                            preview: SharePreview(
                                "PDF Document",
                                image: pdf.thumbnail ?? Image(systemName: "doc.text")
                            )
                        )
                        .labelStyle(.iconOnly)
```